webapp/src/main/resources/public/js/actions/branches/BranchesDataSource.js

```diff
@@ -38,6 +39,20 @@ const BranchesDataSource = {
                 branchStatisticSearcherParameters.empty(false);
             }
 
+            if (branchesSearchParamState.createdBefore) {
+                branchStatisticSearcherParameters.createdBefore(
+                    moment(branchesSearchParamState.createdBefore)
+                        .format('YYYY-MM-DDTHH:mm')
```

Comment:
I preferred to handle using moment because this way we can ensure the format that will be correct both from the input change and the queryString retrieval. This format makes the url cleaner as well; easier to change the link by hand because it won't have all of the escape characters.